rocky/katalogus/templates/clone_settings.html

```diff
@@ -9,7 +9,7 @@ <h2>{% translate "Clone settings" %}</h2>
         This includes both the KAT-alogus settings as well as enabled and disabled plugins.
       {% endblocktranslate %}
         </p>
-        <form action="{% url "katalogus_clone_settings" organization.code %}"
+        <form action="{% url "katalogus_settings" organization.code %}"
               method="post"
               class="horizontal-view help"
               novalidate>
```

Comment:
Is novalidate hier nuttig op? Die skipped namelijk ook de required fields daarmee.